waspc/src/Parser.hs

```diff
@@ -74,8 +79,8 @@ waspParser = do
     -- e.g. check there is only 1 title - if not, throw a meaningful error.
     -- Also, check there is at least one Page defined.
 
-    return $ (Wasp.fromWaspElems waspElems) `Wasp.setJsImports` jsImports
+    return $ Wasp.fromWaspElems waspElems `Wasp.setJsImports` jsImports
```

Comment:
maybe it is clearer with parentheses

Reply:
Maybe, Hlint suggested this so I went with it hm!